src/schemas/json/package.json

```diff
@@ -127,6 +127,26 @@
           "$ref": "#/definitions/packageExportsFallback"
         }
       ]
+    },
+    "fundingUrl": {
+      "type": "string",
+      "format": "uri",
+      "description": "URL to a website with details about how to fund the package."
+    },
+    "fundingWay": {
+      "type": "object",
+      "description": "Used to inform about ways to help fund development of the package.",
+      "properties": {
+        "url": {
+          "$ref": "#/definitions/fundingUrl"
+        },
+        "type": {
```

Comment:
The type property is optional, not required - this is incorrectly telling me my hundreds of projects’ package.jsons are wrong.

Reply:
Sorry, could you provide the related parts of the npm source code?

Reply:
The documentation is irrelevant; "how npm behaves" is the specification, and how it behaves is that the type is optional.

Your editorial opinion on what an author "should" do is valid but also irrelevant. I do this - an object with only a "url" property - on over 300 quite popular npm packages, and vscode shouldn't be complaining about it.

Reply:
You can use [all-repos](https://github.com/asottile/all-repos) to make a batch change over all your affected repos. Anyways, I'm not a maintainer of SchemaStore and I've not strong opinion about what should be validate inside `funding` objects, so feel free to open a PR.

Reply:
Filed #2136.